appveyor.bat

```diff
@@ -12,7 +12,7 @@ echo Building MinGW
 set PATH=C:\mingw-w64\x86_64-7.2.0-posix-seh-rt_v5-rev1\mingw64\bin;%PATH:C:\Program Files\Git\usr\bin;=%
 mkdir build_mingw
 cd build_mingw
-cmake -G "%generator%" -Dprotobuf_BUILD_SHARED_LIBS=%BUILD_DLL% -Dprotobuf_UNICODE=%UNICODE% -Dprotobuf_BUILD_TESTS=0 ../cmake
+cmake -G "%generator%" -Dprotobuf_BUILD_SHARED_LIBS=%BUILD_DLL% -Dprotobuf_BUILD_TESTS=0 ../cmake
```

Comment:
Is this just cleanup?  Or is this a meaningful change?

Reply:
cleanup

Reply:
I didn't even know this file existed tbh, and I'm not familiar with what it's meant for.  When is %UNICODE% set?  I assume that's some environment variable users might have unset?

Reply:
If appveyor is not activated in CI, it is safe to remove it.

Reply:
And `protobuf_UNICODE` is not documented, so it should be safe to ignore.

Reply:
Our cmake documentation is pretty poor, *a lot* of people use our cmake configs as "documentation" and figure out flags from there.  So if this is a behavior change it's not quite "safe"